tomcat/src/main/java/camp/nextstep/http/HttpResponse.java

```diff
@@ -0,0 +1,63 @@
+package camp.nextstep.http;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class HttpResponse {
+
+  private static final String CONTENT_TYPE = "Content-Type";
+  private static final String CONTENT_LENGTH = "Content-Length";
+  private static final String LOCATION = "Location";
+  private static final String CHARSET_UTF8 = ";charset=utf-8";
+  private final HttpStatus status;
```

Comment:
```suggestion
  private static final String CHARSET_UTF8 = ";charset=utf-8";
  
  private final HttpStatus status;
```
상수와 상태값은 구분지어주면 좋습니다 :)